packages/jsii-java-runtime/project/src/test/java/software/amazon/jsii/JsiiObjectMapperTest.java

```diff
@@ -38,18 +41,21 @@ public void testDeserialization() throws Exception {
 
     when((TestEnum) jsiiEngine.findEnumValue("module.Enum#value")).thenReturn(TestEnum.DUMMY);
 
-    final String json =
-        Resources.toString(Resources.getResource(this.getClass(), "complex-callback.json"), Charsets.UTF_8);
-    final TestCallback callback = subject.treeToValue(subject.readTree(json), TestCallback.class);
-
-    assertEquals("CallbackID", callback.getCbid());
-    assertNotNull(callback.getInvoke());
-    if (callback.getCbid() != null) {
-      assertEquals("methodName", callback.getInvoke().getMethod());
-      assertEquals(1337, callback.getInvoke().getArgs().get(0));
-      assertEquals(mockObject1, callback.getInvoke().getArgs().get(1));
-      assertEquals(Instant.ofEpochMilli(1553624863569L), callback.getInvoke().getArgs().get(2));
-      assertEquals(TestEnum.DUMMY, callback.getInvoke().getArgs().get(3));
+    try (final InputStream stream = this.getClass().getResourceAsStream("complex-callback.json");
+         final InputStreamReader reader = new InputStreamReader(stream, Charset.forName("UTF-8"));
```

Comment:
StandardCharsets.UTF_8 is probably preferable here

Reply:
Oh nice! I didn't know this was a thing!